### DISCUSSION TITLE
DB 제약 조건 이름 컨벤션 정하는 것이 좋겠어요

### DISCUSSION THREAD

**Discussion by jyeost** [woowacourse-teams/2023-edonymyeon#396] [백엔드 토론의 장]
@hectick @This2sho @BackFoxx 

db 변경 사항이 많아지면서 이전 디비 스키마를 바꾸는 일이 많은데 기존 JPA로 만든 
fk 제약 조건 이름이 랜덤으로 발생되어서... flyway를 날리기 전에 테스트 서버와 운영서버의 키 이름을 바꾼 뒤 실행해야 했는데여...

새로운 테이블은 만들면서 제약 조건 이름을 직접 지어주니까 어느정도 컨벤션으로 만들어 졌으면 좋겠어요 !
린네상 보니 뽀링키는 `fk_테이블명_필드명` 이렇게 걸고 있는 것 같더라고여!!
저는 인덱스를 이번에 `i_테이블명_필드명` 이렇게 지었습미다!! 

문제 없다면 앞으로 다들 맞춰서 작명해주시길... 🙏🏻

**Comment by BackFoxx** [comment]
동의합니다

**Comment by hectick** [comment]
반영하겠읍니당!
